### PR TITLE
[14.0] [MIG] base_currency_iso_4217: Migration to v14

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26428

please review this migration @joao-p-marques @carlosdauden

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: base_currency_iso_4217/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#. module: base_currency_iso_4217
+#: model:ir.model.fields,field_description:base_currency_iso_4217.field_res_currency__full_name
+msgid "Full name"
+msgstr "Nombre completo"
+
+#. module: base_currency_iso_4217
+#: model:ir.model.fields,help:base_currency_iso_4217.field_res_currency__numeric_code
+msgid "ISO Numeric Code for currency, according to ISO 4217 standard."
+msgstr "Código numérico ISO, como indica el estándar ISO 4217."
```
Comment (joao-p-marques) [inline base_currency_iso_4217/i18n/es.po]:
```suggestion
msgstr "Código numérico ISO, de acuerdo con el estándar ISO 4217."
```

**Review by joao-p-marques — APPROVED** [review]
Code review :+1: 

About the translation, I don't know if this would be a better alternative. Not very important, though.

**Comment by CarlosRoca13** [comment]
I applied the changes @joao-p-marques, thanks for the revision

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**